.github/workflows/bcny-cmake.yml

```
@@ -6,44 +6,88 @@ on:
 jobs:
   build:
     runs-on: swift-build-windows-latest-32-cores
+    outputs:
+      artifact_amd64: ${{ steps.vars.outputs.artifact_amd64 }}
+      artifact_arm64: ${{ steps.vars.outputs.artifact_arm64 }}
+      release: $${{ steps.vars.outputs.release }}
+
+    strategy:
+      matrix:
```

I think that marking this as `fail-fast: false` is a good idea.

---

Done

---

Why? Is it okay for one to fail and the other to succeed? If not then fail fast saves time by canceling in progress work.

---

@compnerd

---

@amonshiz because we can have architecture specific behaviour. Just because the ARM64 build failed doesn't imply that the AMD64 build will fail and vice versa.

---

I see that even if one arch build failed, it's useful to see if the other arch build succeeds or not.